Workflow.yml

```diff
@@ -94,6 +94,8 @@ $graph:
            incorrectly listed as optional.
         * For multi-Process CWL documents, if no particular process is named then
           the process with the `id` of `#main` is chosen.
+        * [Made explicit](#Generic_execution_process) that the `$namespaces` of
```

Comment:
Maybe tweak this to say the $namespaces specifically come from the top level document